File: Suuretu03.java
import java.util.Scanner;

public class Suuretu03 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Input n
        int n = scanner.nextInt();

        // Input n numbers01
        int[] numbers01 = new int[n];
        for (int i = 0; i < n; i++) {
            numbers01[i] = scanner.nextInt();
        }

        // Input m
        int m = scanner.nextInt();

        // Input n numbers02
        int[] numbers02 = new int[m];
        for (int i = 0; i < m; i++) {
            numbers02[i] = scanner.nextInt();
        }

        for (int i = 0; i < m; i++) {
            int index = numbers02[i] - 1;
            if (index < numbers01.length) {
                System.out.println(numbers01[index]);
            } else {
                System.out.println("Ten");
            }
        }
    }
}

File: Suuretu03.c
#include <stdio.h>

int main() {
    // Input n
    int n;
    scanf("%d", &n);

    // Input n numbers01
    int numbers01[n];
    for (int i = 0; i < n; i++) {
        scanf("%d", &numbers01[i]);
    }

    // Input m
    int m;
    scanf("%d", &m);

    // Input n numbers02
    int numbers02[m];
    for (int i = 0; i < m; i++) {
        scanf("%d", &numbers02[i]);
    }

    for (int i = 0; i < m; i++) {
        int index = numbers02[i] - 1;
        if (index < n) {
            printf("%d\n", numbers01[index]);
        } else {
            printf("Ten\n");
        }
    }

    return 0;
}

